fundingmanager.go

```diff
@@ -454,11 +472,6 @@ func (f *fundingManager) Start() error {
 		f.newChanBarriers[chanID] = make(chan struct{})
 		f.barrierMtx.Unlock()
 
-		// Set up a localDiscoverySignals to make sure we finish sending
-		// our own fundingLocked and channel announcements before
-		// processing a received fundingLocked.
-		f.localDiscoverySignals[chanID] = make(chan struct{})
```

Comment:
Why was this removed?

Reply:
If I'm not mistaken, the `localDiscoverySignal` has the purpose of making sure we have seen the funding tx confirm on-chain before handling an incoming `fundingLocked`. In this case the channels already have their funding txs confirmed, we just need to send the fundinglocked and channel announcements, hence we won't need the `localdiscoverysignal`. This was added by me by mistake earlier, and became apparent here since we now will wait six confirmations before sending the channel announcements, and previously we wouldn't handle the incoming fundinglocked before this was done.